doc/tutorial/statistical_inference/supervised_learning.rst

```diff
@@ -543,7 +496,7 @@ creating a decision energy by positioning *kernels* on observations:
 
 .. image:: /auto_examples/datasets/images/sphx_glr_plot_iris_dataset_001.png
```

Comment:
Can we leave this image within the "exercise"?
Or maybe put it after, with a title. Right now it's not obvious what this refers to.